### PR TITLE
Support Vector Machine Base issue#25

### PR DESCRIPTION
**Pull request by ptnv-s**
Implementing a Linear Kernel SVM . The code works correctly and gives a accuracy of 88% in titanic dataset of kaggle.

### CODE REVIEW

**Comment by ashwani-rathee** [comment]
@ptnv-s Can you add numpy as essential import and also provide code documentation(in .py file only) that kind of give a example on how to test it?

**Comment by ptnv-s** [comment]
> @ptnv-s Can you add numpy as essential import and also provide code documentation(in .py file only) that kind of give a example on how to test it?

I want to try, can i work on it and submit by 2 days?

**Comment by ashwani-rathee** [comment]
@ptnv-s works  :)

**Comment by ptnv-s** [comment]
for the code documentation i.e. example of problem statement , is it acceptable to work on a dataset which requires bit of EDA before applying SVM (like Titanic)?

**Comment by ptnv-s** [comment]
> > @ptnv-s Can you add numpy as essential import and also provide code documentation(in .py file only) that kind of give a example on how to test it?
> 
> I want to try, can i work on it and submit by 2 days?

Done @ashwani-rathee

**Comment by ptnv-s** [comment]
@ashwani-rathee  Any update . Does anything else needs to be done?

**Comment by ashwani-rathee** [comment]
Can you remove  the test documentation or include it in main implementation and then comment it out?

**Comment by ashwani-rathee** [comment]
Other than that looks good to me @geekquad

**Comment by ptnv-s** [comment]
@ashwani-rathee could you check the recent update and tell if anything needs to be done?

**Comment by ptnv-s** [comment]
@geekquad  could you check the recent update and tell if anything needs to be done?

File: Support Vector Machine/SVM_Linear_Kernal_&_documentation.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    sum = 0
+    for i in range(len(prediction)):
+        if y_test[i] == prediction[i]:
+            sum = sum + 1
+            acc = sum / len(prediction)
+
+    return acc
+
+
+def initialization():
```
Comment (ashwani-rathee) [inline Support Vector Machine/SVM_Linear_Kernal_&_documentation.py:51]: @ptnv-s after this,the content is redundant

Comment (ashwani-rathee) [reply Support Vector Machine/SVM_Linear_Kernal_&_documentation.py:51]: please comment it out

Comment (ptnv-s) [reply Support Vector Machine/SVM_Linear_Kernal_&_documentation.py]: @ashwani-rathee  did it in latest commit ,could you review it please

**Comment by ptnv-s** [comment]
@geekquad @ashwani-rathee  any update regarding PR?

**Review by geekquad — APPROVED** [review]
Amazing work @ptnv-s.

**Comment by ptnv-s** [comment]
> Amazing work @ptnv-s.

Thanks a lot @geekquad .Looking forward to contribute more.

**Pull request merged**